packages/cloudvision-connector/src/Wrpc.ts

```diff
@@ -374,7 +380,8 @@ export default class Wrpc {
    * Send a service request command to the CloudVision API
    */
   public requestService(request: ServiceRequest, callback: NotifCallback): string | null {
-    return this.callCommand(SERVICE_REQUEST, request, callback);
+    const result = this.callCommand(SERVICE_REQUEST, request, callback);
+    return result ? result.token : null;
```

Comment:
did you try to cancel these requests? I think they should also be cancellable

Reply:
i didnt. we can add that later if needed.